packages/upgrade/transforms/__tests__/enable-v12-tile-radio-icons-test.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
To match other files in this directory

```suggestion
/**
 * Copyright IBM Corp. 2025
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

'use strict';
```